index.html

```diff
@@ -6,18 +6,18 @@
     <title>Radium - A React Component Styling Library</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="/static/favicon.ico" type="image/x-icon">
-    <base href="/" />
     <!-- Fonts  -->
     <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400,600" rel="stylesheet" type="text/css">
     <link href="/static/fonts.css" rel="stylesheet" type="text/css">
-    <!-- Syntax highlighting theme -->
-    <link href="/static/prism-estuary.light.css" rel="stylesheet" type="text/css">
-    <!-- <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.4.1/themes/prism.min.css" rel="stylesheet" type="text/css"> -->
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.0.0/codemirror.min.css"/>
+    <link href="/static/prism-github-dark.css" rel="stylesheet" type="text/css">
 </head>
 <body>
-  <div id="content">
+  <div id="content" style="height: 100vh">
     <p style="text-align: center;">Oops! Something is broken or JavaScript is not enabled.</p>
   </div>
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.0.0/codemirror.min.js"></script>
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.0.0/mode/javascript/javascript.min.js"></script>
```

Comment:
I believe `codemirror.js` is bundled with `component-playground`, since it moved to `react-code-mirror`. See https://github.com/FormidableLabs/formidable-landers/issues/106